src/read.c

```diff
@@ -4677,6 +4677,33 @@ static avifResult avifMetaFindAlphaItem(avifMeta * meta,
     return AVIF_RESULT_OK;
 }
 
+static avifResult avifReadColorNclxProperty(const avifPropertyArray * properties,
```

Comment:
We should document what this function does to the output parameters when it returns `AVIF_RESULT_OK`.

Similarly, the `avifReadColorProperties()` function at line 4707 should be documented.

Reply:
Done.